eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -80,14 +80,21 @@ steps:
       dotnet tool restore
   condition: "and(not(eq(variables['IsOfficialBuild'], 'true')), eq(variables['BuildRTM'], 'true'))"   #skip this task for nonRTM in private build
 
+# NuGet.Client's official repo is on github, hence the source link metadata should use the github URL.
+# However, our official builds are built from a mirror in Azure DevOps, hence without any extra help, the SourceLink.GitHub package
+# won't be able to determine the URL to embed in the pdbs.
+# Therefore, when the git origin remote is not github (official builds), we need to add the GitHub repo URL as a remote, and
+# tell SourceLink.GitHub what that remote name is
```

Comment:
Thanks for this comment Andy. So, If I understand correctly `SourceLink.GitHub` reads existing `remotes` and choose the url which has `github.com` to embed in pdbs.